server/sonar-server/src/main/java/org/sonar/server/ui/ws/ComponentAction.java

```diff
@@ -281,7 +290,7 @@ private void writeBreadCrumbs(JsonWriter json, DbSession session, ComponentDto c
 
     for (ComponentDto c : breadcrumb) {
       json.beginObject()
-        .prop("key", c.getDbKey())
+        .prop("key", c.getKey())
```

Comment:
We should ask @stas-vilchik if the branch field should now also be returned

Reply:
@julienlancelot Stas confirmed he doesn't need it.